test/unit/org/apache/cassandra/db/ScrubTest.java

```diff
@@ -747,33 +746,19 @@ public void testFilterOutDuplicates() throws Exception
         assertEquals(0, rs.size());
     }
 
-    @Test
-    public void testToolTestingEnvSetup()
-    {
-        createKeyspace(KEYSPACE2,
-                       KeyspaceParams.simple(1),
-                       standardCFMD(KEYSPACE2, CF),
-                       standardCFMD(KEYSPACE2, CF2),
-                       standardCFMD(KEYSPACE2, CF3),
-                       standardCFMD(KEYSPACE2, CF4),
-                       counterCFMD(KEYSPACE2, COUNTER_CF));
-
-        System.setProperty(org.apache.cassandra.tools.Util.ALLOW_TOOL_REINIT_FOR_TEST, "true"); // Necessary for testing
-    }
-
     @Test
     public void testScrubOneRowWithTool()
     {
-        CompactionManager.instance.disableAutoCompaction();
-        Keyspace keyspace = Keyspace.open(KEYSPACE2);
+        String ksName = toolTestingSetup();
+        Keyspace keyspace = Keyspace.open(ksName);
         ColumnFamilyStore cfs = keyspace.getColumnFamilyStore(CF);
         cfs.clearUnsafe();
```

Comment:
Do we still need the calls to {{clearUnsafe}} in the tool tests now that they are using a dedicated keyspace?

Reply:
Some don't call clearUnsafe(), some call it mid-method, etc I remove the calls from the tool based tests, left the rest alone and triggered a run in the multiplexer. Hope it's ok.

Reply:
The multiplexer run looks ok.